medcat/utils/relation_extraction/llama/model.py

```diff
@@ -0,0 +1,191 @@
+import logging
+from typing import Any, Optional, Tuple
+import torch
+from torch import nn
+from transformers.models.llama import LlamaModel, LlamaConfig
+
+from medcat.config_rel_cat import ConfigRelCAT
+from medcat.utils.relation_extraction.ml_utils import create_dense_layers, get_annotation_schema_tag
+from medcat.utils.relation_extraction.models import Base_RelationExtraction
+
+
+class LlamaModel_RelationExtraction(Base_RelationExtraction):
```

Comment:
why the underscore in the name, just `BaseRelationExtraction` and `LlamaRelationExtraction` look fine?

Reply:
I just moved the classes around - didn't change the names. That's what the convention has been since #173